cog/daily_charge.py

```
@@ -23,95 +23,241 @@
+                        author,
+                        last_charge,
+                        created_time,
+                        cursor,
+                        user_data.charge_combo,
+                        user_data.point,
+                        is_forgivable=True,
+                    )
+
+                    # TODO: send the message together, or there may have problem about send but not modify
+                    embed = self.embed_successful(
+                        user_data.point, user_data.charge_combo, author
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)